src/web/resources/report.js

```diff
@@ -408,7 +406,9 @@ const CostAccuracy = new Component('#cost-accuracy', {
                 let accuracy = 100*(1 - d[1]/bits);
                 let speedup = initial_pt[0]/d[0];
                 return Element("tr", [
-                    Element("th", "Alternative " + (i + 1)),
+                    Element("th",
+                        Element("a", { href: "#alternative" + (i + 1)},
+                                "Alternative " + (i + 1))),
```

Comment:
I don't think this is right. This code iterates through `rest_pts`, which was sorted previously, so it's not in the same order as the ones below.

Reply:
Yesterday it wasn't. After merging main it looks to line up.

Reply:
@cccfire Yes they just happen to be sorted correctly right now. I don't know of a way to enforce that they are sorted the same as half is done in Racket and half in JS. Maybe this feature shouldn't be merged as it may be easy to accidentally broken. Unless I can think of a test for this, but maybe that's not worth it?

Reply:
If you've confirmed that they're sorted the same, let's merge it. Yes, it's brittle, and yes, long term we might break it, but at least in my mind the ideal eventual path is to replace / converge with Odyssey anyway.